plugins/example-events-backend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@internal/plugin-example-events-backend",
```

Comment:
I got inspired by `example-todo-list`, etc. At the end, it is targeting the example backend (`packages/backend`).

`events-backend-module-testing` has a bit of an overlap with `events-backend-test-utils` -- I think it will be hard to understand the difference.

A counter proposal could be to name it `events-backend-module-example` or so.

What do you think @freben ?

Reply:
The "example" name in those packages is only to highlight that they're serving as an example plugin bundle, rather than being an actual production thing. Effectively, its kinda like the plugin name in that case is `example-todo-list`, not that `example` is a word with standalone meaning. Note that it has a `X-backend` and `X-common` next to it too, just like everything else. Its backend package is of role `backend-plugin`, while yours is `backend-plugin-module` plus your readme says "This plugin is a module for the `events-backend` backend plugin" which really makes it seem like a regular module albeit perhaps with a pretty specific purpose.

So regarding naming: it exports just a single subscriber class at this point, one which is useful for anyone who just wants to do some quick testing. I'm rather thinking, should that class be part of the base offering? Could it even be exported by e.g. `events-backend-node` as a utility class that you can instantiate and inject "by hand" in your backend init code because you're trying stuff out or trying to find problems?

Reply:
The only issue with adding it to `events-backend-node` is that the class implements `EntityProvider`, too, so that we can use it at `packages/backend/src/plugins/catalogEventBasedProviders.ts` instead of only at `packages/backend/src/plugins/events.ts`. This was the only reason for implementing `EntityProvider`, it actually doesn't do anything.

This means, we would need to add a dependency to `@backstage/plugin-catalog-node`.
Maybe not a big deal though.

Other than that adding it to `events-backend-node` would sound good to me.

Reply:
plan after discussing on Discord

1. add debug logging to the broker's publish method
2. add `DemoEventBasedEntityProvider` directly to the `catalogEventBasedProviders.ts` file at the example backend (info logging only)
3. remove `plugins/example-events-backend/`